src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java

```diff
@@ -76,19 +75,22 @@
  * for non-diploid organisms.</p>
  *
  */
-@CommandLineProgramProperties(summary = "Perform joint genotyping on one or more samples pre-called with HaplotypeCaller", oneLineSummary = "Perform joint genotyping on one or more samples pre-called with HaplotypeCaller", programGroup = VariantProgramGroup.class)
+@CommandLineProgramProperties(summary = "Perform joint genotyping on one or more samples pre-called with HaplotypeCaller",
```

Comment:
"Perform joint genotyping on one or more samples pre-called with HaplotypeCaller" --> "Perform joint genotyping on a HaplotypeCaller GVCF callset"

Reply:
Let's avoid `pre-called`

Reply:
This summary is old and now inaccurate.  The GATK4 tool cannot take multiple HaplotypeCaller GVCFs.  "Perform joint genotyping on a single-sample GVCF from HaplotypeCaller or a multi-sample GVCF from CombineGVCFs"

Reply:
I will add a caveat "Cannot take multiple GVCF files in one command."

Reply:
Amend summary to "Perform joint genotyping on a single-sample GVCF from HaplotypeCaller or a multi-sample GVCF from CombineGVCFs or a GenomicsDB datastore"